docs/pages/modules/get-started.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Get Started
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+import { Terminal } from '~/ui/components/Snippet';
+
+## Create a new module
+
+To create a new Expo module from scratch, just run the `create-expo-module` script as below. The script will ask you a few questions and then generate the native Expo module along with the example app for iOS and Android that uses your new module.
+
+<Terminal cmd={[`$ npx create-expo-module`]} />
+
+## Set up a library as an Expo module
+
+You may want to use the Expo Modules API in existing React Native libraries, for example with [AppDelegate Subscribers](./appdelegate-subscribers.mdx) you can hook into `AppDelegate` methods without requiring developers to copy any code over to their own `AppDelegate`. This is particularly useful to add seamless support for Expo managed projects to a library. The following steps will set up your existing React Native library to have access to the Expo Modules API.
```

Comment:
```suggestion
You can use the Expo Modules API in existing React Native libraries. For example, with [AppDelegate Subscribers](./appdelegate-subscribers.mdx) you can hook into `AppDelegate` methods without requiring developers to copy any code to their project's `AppDelegate`. This is particularly useful to add seamless support for Expo managed projects to a library. The following steps will set up your existing React Native library to access the Expo Modules API.
```

As per the [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#avoiding-using-outdated-workflows), let's try to avoid the mention of "managed" (As per my understanding, I assume, here you meant the "projects in the managed workflow"?).

Also, to me, "Expo managed projects to a library" phrase sounds confusing. I don't understand what "projects to a library" mean in this context. Can you please explain that to me?
From my understand, I think you mean to say, libraries that can run in Expo projects in general (whether they are developing using Expo Go or Development builds?)

Reply:
Yeah, I meant "projects in the managed workflow". This feature is like config plugins — it's more useful in the managed workflow, but also works in bare so I think we can just remove this phrase if it sounds confusing.

Reply:
I entirely removed that part about AppDelegate Subscribers, it was a bit out of context.